CHANGELOG.rst

```diff
@@ -3,6 +3,19 @@ Release Notes
 
 Changes are ordered reverse-chronologically.
 
+0.9.1
+-----
+
+ - Fixed regression that caused very slow imports of large channels
+ - Adds new 'import users' command to the command-line
+ - Various consistency and layout updates
+ - Exercises with an error no longer count as 'correct'
+ - Fixed issue with password-less sign-on
+ - Various other fixes
```

Comment:
Reference `The remove option in Lesson does not work all the time` #3515

Reply:
thanks